tests/unit/lms/data_tasks/report/create_from_scratch/test_01_functions/test_01_date_functions.py

```diff
@@ -9,29 +9,17 @@
 
 
 class TestDateFunctions:
-    NOW = datetime(2025, 3, 6, tzinfo=UTC)  # We hardcode the date to avoid flakiness
-    # 2024 was a leap year so the math accounts for that 366 days
-    ONE_YEAR_AGO = (NOW - timedelta(days=366)).date()
-    TWO_YEARS_AGO = (NOW - timedelta(days=366 + 365)).date()
+    # We hardcode the date to avoid flakiness
+    NOW = date(2025, 3, 6)
 
     @pytest.mark.usefixtures("with_date_functions")
     @pytest.mark.parametrize(
         "timescale,value,expected",
         (
-            ("trailing_year", "'2025/03/06'::date", ONE_YEAR_AGO),
```

Comment:
Ignoring trailing_year completely for now, it uses the current date internally in postgres so it's not possible to easily mock it.